1.8/administration/access-node/tunnel.md

```diff
@@ -79,7 +79,6 @@ DC/OS Tunnel provides you with full access to the DNS, masters, and agents from
           <td>
               <ul>
                   <li>Requires application configuration</li>
-                  <li>Only supports HTTP</li>
```

Comment:
I don't this should be removed, only changed to HTTP/HTTPS, since it doesn't support other protocols.